src/store/metadata/stub/content/editions/domestic_energy_performance_certi_existing_dwellings_by_eng_eff_rating.json

```diff
@@ -0,0 +1,97 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/DLUHC/editions",
+    "@type": "hydra:Collection",
+    "title": "Domestic Energy Performance Certificates for existing dwellings",
+    "editions": [
+        {
+            "@id": "https://staging.idpd.uk/datasets/domestic_energy_performance_certi_existing_dwellings_by_eng_eff_rating_2023_07/editions/2023-10",
```

Comment:
Change this to use the unique identifier, along with other instances throughout.

Reply:
used `epcs-for-existing-domestic-properties` throughout